src/components/Form/index.js

```diff
@@ -0,0 +1,38 @@
+import Input from "../Input";
+import Button from "../Button";
+import "./Form.css";
+
+import React, { useState, useEffect } from "react";
+
+export default function Form({ label, handleAddingItem }) {
+  // Creador de ToDos
+  function Task(title = "No title...", completed = false) {
```

Comment:
Prefer the const a = () => 
syntax for function handlers inside a component instead of using again Function syntax (like the component declaration)